features/Screen-Space Shadows/Shaders/ScreenSpaceShadows/RaymarchCS.hlsl

```diff
@@ -70,15 +19,28 @@ float InterleavedGradientNoise(float2 uv)
 	return frac(magic.z * frac(dot(uv, magic.xy)));
 }
 
-float ScreenSpaceShadowsUV(float2 texcoord, float3 lightDirectionVS)
+float GetScreenDepth(float depth)
+{
+	return (CameraData.w / (-depth * CameraData.z + CameraData.x));
+}
+
+float GetScreenDepth(float2 uv, uint a_eyeIndex)
 {
+	uv = ConvertToStereoUV(uv, a_eyeIndex);
+	float depth = GetDepth(uv);
+	return GetScreenDepth(depth);
+}
+
+float ScreenSpaceShadowsUV(float2 stereoTexCoord, uint eyeIndex)
+{
+	float2 texcoord = ConvertFromStereoUV(stereoTexCoord, eyeIndex);
 	// Ignore the sky
-	float startDepth = GetDepth(texcoord);
+	float startDepth = GetDepth(stereoTexCoord);
```

Comment:
We try to maintain transparent compatibility with flat. In other words, flat should feel comfortable modifying the code for improvements. See how the convert to stereo functions work.

Reply:
Reverted the name and only added conversion in case it was needed (same logically just less changes required)